// @capacitor-community/capacitor-googlemaps-native/dist/esm/web.js
import { WebPlugin } from '@capacitor/core';
export class CapacitorGoogleMapsWeb extends WebPlugin {
    constructor() {
        super({
            name: 'CapacitorGoogleMaps',
            platforms: ['web'],
        });
    }
    create(_options) {
        throw new Error('Method not implemented.');
    }
    initialize(_options) {
        throw new Error('Method not implemented.');
    }
    addMarker(_options) {
        throw new Error('Method not implemented.');
    }
    removeMarker(_options) {
        throw new Error('Method not implemented.');
    }
    setCamera(_options) {
        throw new Error('Method not implemented.');
    }
    setMapType(_options) {
        throw new Error('Method not implemented.');
    }
    setIndoorEnabled(_options) {
        throw new Error('Method not implemented.');
    }
    setTrafficEnabled(_options) {
        throw new Error('Method not implemented.');
    }
    accessibilityElementsHidden(_options) {
        throw new Error('Method not implemented.');
    }
    padding(_options) {
        throw new Error('Method not implemented.');
    }
    clear() {
        throw new Error('Method not implemented.');
    }
    close() {
        throw new Error('Method not implemented.');
    }
    hide() {
        throw new Error('Method not implemented.');
    }
    show() {
        throw new Error('Method not implemented.');
    }
    settings(_options) {
        throw new Error('Method not implemented.');
    }
    reverseGeocodeCoordinate(_options) {
        throw new Error('Method not implemented.');
    }
    enableCurrentLocation(_options) {
        throw new Error('Method not implemented.');
    }
    myLocation(_options) {
        throw new Error('Method not implemented.');
    }
    viewBounds() {
        throw new Error('Method not implemented.');
    }
    setMapStyle(_options) {
        throw new Error('Method not implemented.');
    }
    addPolyline(_options) {
        throw new Error('Method not implemented.');
    }
    addCircle(_options) {
        throw new Error('Method not implemented.');
    }
    addPolygon(_options) {
        throw new Error('Method not implemented.');
    }
    setOnMarkerClickListener() {
        throw new Error('Method not implemented.');
    }
    setOnMapClickListener() {
        throw new Error('Method not implemented.');
    }
    setOnPoiClickListener() {
        throw new Error('Method not implemented.');
    }
    requestLocationPermission() {
        throw new Error('Method not implemented.');
    }
    setOnMyLocationClickListener() {
        throw new Error('Method not implemented.');
    }
    setOnMyLocationButtonClickListener() {
        throw new Error('Method not implemented.');
    }
}
